modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsMapFieldPropertyValue.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Configure Behavior for paragraphs.
```

Comment:
This comment needs updating to reflect what it does.

Reply:
Done